### PR TITLE
ENCD-6227-immune-slims

### PR DESCRIPTION
**Pull request by jenjou**

### CODE REVIEW

File: src/encoded/schemas/award.json [hunk 1]
```diff
@@ -106,6 +106,7 @@
                             "pooled clone sequencing",
                             "PRO-cap",
                             "PRO-seq",
+                            "proliferation CRISPR screen",
```
Comment (gabdank) [inline src/encoded/schemas/award.json:111]: should not we include FACS CRISPR screen and FlowFISH CRISPR screens?

File: src/encoded/schemas/changelogs/chip_library_quality_metric.md [hunk 2]
```diff
@@ -8,4 +8,4 @@
 * Added *Capture Hi-C* and *single-nucleus RNA-seq* to *assay_term_name* enum
 * Added *GRO-cap*, *GRO-seq*, and *long read single-cell RNA-seq* to *assay_term_name* enum;  *single-nucleus RNA-seq* and *genotyping by high throughput sequencing assay* were removed and remapped to *single-cell RNA sequencing assay* and *whole genome sequencing assay* respectively
 * Removed *single-cell ATAC-seq* from *assay_term_name* enum and remapped to *single-nucleus ATAC-seq*
-* Added *LC/MS label-free quantitative proteomics*, *LC-MS/MS isobaric label quantitative proteomics*, and *Ribo-seq* to *assay_term_name* enum. *assay_term_name* enum *Capture Hi-C* was changed to *capture Hi-C*
+* Added *LC/MS label-free quantitative proteomics*, *LC-MS/MS isobaric label quantitative proteomics*, and *Ribo-seq* to *assay_term_name* enum. *assay_term_name* enum *Capture Hi-C* was changed to *capture Hi-C*
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/chip_library_quality_metric.md]: not sure what changed here?

**Review by gabdank — APPROVED** [review]

File: src/encoded/schemas/award.json (hunk 1 above)
Comment (jenjou) [reply src/encoded/schemas/award.json:111]: Unsure what happened but I tried to push again (despite not having a new commit), and I am able to see the addition of FACS and FlowFISH now.

Comment (gabdank) [reply src/encoded/schemas/award.json:111]: yep

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — DISMISSED** [review]

**Pull request merged**